package.json

```diff
@@ -56,7 +56,8 @@
     "babel-runtime": "^6.22.0",
     "classnames": "^2.1.5",
     "debug": "^2.6.3",
-    "lodash": "^4.17.2"
+    "lodash": "^4.17.2",
+    "prop-types": "^15.5.0"
```

Comment:
I'm not sure about this, however, I'm followed [recommendation](https://github.com/reactjs/prop-types#how-to-depend-on-this-package) there.